src/Components/SideMenu/SideMenu.css

```diff
@@ -110,13 +111,6 @@ select.sort-schema-options {
   margin-right: 2%;
 }
 
-.table-view-controls {
-  display: inline-flex;
-  width: 100%;
-  height: 28px;
-  margin-top: 50px;
-}
-
```

Comment:
@Synicix What's the reason why this was removed?

Reply:
Was trying to split the CSS since the original commit from Maho on this side menu had the entire schema and table list into one giant file. That didn't sit well with me so I split the code apart and rewrote it to make it eaiser to work on for the future.

The CSS for this is still a giant mess, and a relic from that era. I never finished fixing the CSS cause I was focus on more user visable problems first.